### PR TITLE
2FA: Add notes about the existence of 2FA PC apps

### PR DESCRIPTION
**Pull request by DandelionSprout**
Since Authy and various other 2FA apps have been available on PC operative systems for some years by now, I figured out that it's time to acknowledge them in the text strings and menus and such.

### CODE REVIEW

File: client/me/security-2fa-enable/index.jsx [hunk 1]
```diff
@@ -218,7 +218,7 @@ class Security2faEnable extends React.Component {
 			<div className="security-2fa-enable__time-code-block">
 				<p className="security-2fa-enable__time-instruction">
 					{ this.props.translate(
-						'Enter this time code into your mobile app. {{toggleMethodLink}}Prefer to scan the code?{{/toggleMethodLink}}',
+						'Enter this time code into your authenticator app. {{toggleMethodLink}}Prefer to scan the code?{{/toggleMethodLink}}',
```
Comment (a8ci18n) [inline client/me/security-2fa-enable/index.jsx]: :ok: This change is safe, we'll keep the existing 20 translations.

File: client/me/security-2fa-enable/index.jsx [hunk 2]
```diff
@@ -198,7 +198,7 @@ class Security2faEnable extends React.Component {
 			<div className="security-2fa-enable__qr-code-block">
 				<p className="security-2fa-enable__qr-instruction">
 					{ this.props.translate(
-						"Scan this QR code with your mobile app. {{toggleMethodLink}}Can't scan the code?{{/toggleMethodLink}}",
+						"Scan this QR code with your authenticator app. {{toggleMethodLink}}Can't scan the code?{{/toggleMethodLink}}",
```
Comment (a8ci18n) [inline client/me/security-2fa-enable/index.jsx]: :ok: This change is safe, we'll keep the existing 19 translations.

File: client/me/security-2fa-setup-backup-codes/index.jsx [hunk 3]
```diff
@@ -97,7 +97,7 @@ class Security2faSetupBackupCodes extends React.Component {
 				<Security2faProgress step={ 3 } />
 				<p>
 					{ this.props.translate(
-						'Backup codes let you access your account if your phone is ' +
+						'Backup codes let you access your account if your unit is ' +
```
Comment (a8ci18n) [inline client/me/security-2fa-setup-backup-codes/index.jsx]: Hi! I've found a possible matching string that has already been translated **29** times:
`translate( 'Backup codes let you access your account if your phone is lost, stolen, or if you run it through the washing machine and the bag of rice trick doesn't work.' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/me/security-2fa-enable/index.jsx [hunk 4]
```diff
@@ -198,7 +198,7 @@ class Security2faEnable extends React.Component {
 			<div className="security-2fa-enable__qr-code-block">
 				<p className="security-2fa-enable__qr-instruction">
 					{ this.props.translate(
-						"Scan this QR code with your mobile app. {{toggleMethodLink}}Can't scan the code?{{/toggleMethodLink}}",
+						"Scan this QR code with your 2FA app. {{toggleMethodLink}}Can't scan the code?{{/toggleMethodLink}}",
```
Comment (a8ci18n) [inline client/me/security-2fa-enable/index.jsx]: :ok: This change is safe, we'll keep the existing 19 translations.

Comment (akirk) [reply client/me/security-2fa-enable/index.jsx]: For consistency this should rather also be `authenticator app`, no?

**Comment by DandelionSprout** [comment]
After I failed the CircleCI check the first time, as it complained about being longer than 140 characters (which it wasn't really), I tried to shorten that line to try to pass the check. But since the check also failed the 2nd time around, I can indeed just as well change it back to "authenticator app" right now.

**Comment by DandelionSprout** [comment]
So, how do you want to write such a note to prevent the confusion of authenticator apps and password managers? I've of course granted maintainer editing rights to this pull requests (to the best of my knowledge), if you were to want to do it yourself.

**Comment by DandelionSprout** [comment]
I think I've allowed maintainer editing rights to this pull request, if you were to want to add the note about the password/2FA storage concern yourself.

**Comment by akirk** [comment]
Thanks, I'll look for a good place and will try to add it to your repo, hence to this PR.

**Comment by akirk** [comment]
Ok, I have added a message and changed the text of the first paragraph a little so that it now looks like this:
screen shot 2017-12-01 at 11 35 15

(btw using the `master` branch on your repo is not a good idea for future pull requests but it doesn't matter for now)

**Comment by DandelionSprout** [comment]
I saw a bit too late in your guidethrough that I was supposed to create a separate patch branch, but then I know that for anything I'd do in the future, on WordPress or otherwise. :)

File: client/me/security-2fa-initial-setup/index.jsx [hunk 5]
```diff
@@ -36,11 +36,23 @@ class Security2faInitialSetup extends React.Component {
 			<div>
 				<p>
 					{ this.props.translate(
-						'Two-Step Authentication adds an extra layer ' +
-							'of security to your account. Once enabled, logging in to ' +
-							'WordPress.com will require you to enter a unique passcode ' +
-							'generated by an app on your mobile device or sent via text ' +
-							'message, in addition to your username and password.'
+						'Two-Step Authentication (commonly denoted as 2FA) adds an extra ' +
```
Comment (a8ci18n) [inline client/me/security-2fa-initial-setup/index.jsx]: Hi! I've found a possible matching string that has already been translated **28** times:
`translate( 'Two-Step Authentication adds an extra layer of security to your account. Once enabled, logging in to WordPress.com will require you to enter a unique passcode generated by an app on your mobile device or sent via text message, in addition to your username and password.' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/20376)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

File: client/me/security-2fa-enable/index.jsx [hunk 6]
```diff
@@ -218,7 +218,8 @@ class Security2faEnable extends React.Component {
 			<div className="security-2fa-enable__time-code-block">
 				<p className="security-2fa-enable__time-instruction">
 					{ this.props.translate(
-						'Enter this time code into your mobile app. {{toggleMethodLink}}Prefer to scan the code?{{/toggleMethodLink}}',
+						'Enter this time code into your authenticator app. {{toggleMethodLink}}Prefer to ' +
```
Comment (a8ci18n) [inline client/me/security-2fa-enable/index.jsx]: :ok: This change is safe, we'll keep the existing 20 translations.

**Comment by apeatling** [comment]
@akirk Is there anything here that we can merge and not block on a full review?

**Comment by alisterscott** [comment]
This branch needs a rebase. Is this PR still required?

**Comment by alisterscott** [comment]
Closing due to inactivity - please reopen if necessary

**Pull request closed without merging**